tasks.py

```diff
@@ -89,21 +89,25 @@ def lint(c):
     c.run("flake8 django-admin-fast-search tests")
 
 
-@task(help={'bumpsize': 'Bump either for a "feature" or "breaking" change'})
-def release(c, bumpsize=''):
+@task
+def release(c):
     """
     Package and upload a release
```

Comment:
do we also create a github release here or only pypi?

Reply:
We have to publish a new tag on github, can do a release as well. But essentially publishing a tag is what will trigger the release on PyPi